src/firefly/js/ui/ImageSelect.jsx

```diff
@@ -16,7 +16,7 @@ import {dispatchComponentStateChange} from '../core/ComponentCntlr.js';
 import {updateSet} from '../util/WebUtil.js';
 
 import './ImageSelect.css';
-
+import infoIcon from 'html/images/info-16x16.png';
```

Comment:
The icon is fine, except it too small or badly re-rendered.  I can't make out the `i` in the icon.